src/classes/Test_DMLManager.cls

```diff
@@ -491,7 +491,58 @@ private class Test_DMLManager {
 			//expected
 			System.assert(dmle.getMessage().contains('An existing record could not be found'));
 		}
-	} 
+	}
+
+	@IsTest
+	static void dmlOperation_When_CustomInertDmlOperationsAreSpecified_Expect_NoChangesWithinServer() {
+
+		Contact ctc = new Contact();
+		ctc.FirstName = 'Jeremiah';
+		ctc.LastName = 'Matthews';
+
+		// ------ First let's clearly assert the default DML operation is working ------
+		DMLManager.insertAsUser(ctc);
+		
+		String query = 'select Id from Contact where FirstName=\'' + ctc.FirstName + '\' and LastName=\'' + ctc.LastName + '\'';
```

Comment:
Wondering why you chose to use dynamic SOQL instead of a static query, here and elsewhere.